assets/stylesheets/common/common.scss

```diff
@@ -422,7 +422,7 @@ body.user-preferences-activity-pub-page {
   align-items: center;
   justify-content: space-between;
 
-  @media all and (max-width: 400px) {
+  @media all and (width <= 400px) {
```

Comment:
Could probably switch this to use the new viewport mixin.

Reply:
That would be a change in design/behaviour, so probably best done in a separate PR.